pom.xml

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>hadoop-router-fs</name>
+    <groupId>io.lakefs</groupId>
+    <artifactId>hadoop-router-fs</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1.0</version>
+
+    <licenses>
+        <license>
+            <name>apache2</name>
+            <url>http://www.apache.org/licenses/</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Treeverse lakeFS dev</name>
+            <email>support@treeverse.io</email>
+            <organization>lakefs.io</organization>
+            <organizationUrl>https://lakefs.io</organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <url>https://github.com/treeverse/hadoop-router-fs</url>
+        <connection>scm:git:git://github.com/treeverse/hadoop-router-fs.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/treeverse/hadoop-router-fs.git</developerConnection>
+    </scm>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <!--The common Hadoop version for Spark 2.4.7 and 3.0.1, which lakeFS supports-->
+        <hadoop.version>2.7.7</hadoop.version><!--MARKER: keep this comment here, we update the version with the lakeFS version when on deploy-->
```

Comment:
This is part of an automatic replacement added by @nopcoder .  Will we need it?

Reply:
Not necessarily, I removed it thanks